// @primer/primitives/tokens-v2-private/js/tokens/functional/typography/typography.js
/**
 * Do not edit directly
 * Generated on Thu, 21 Jul 2022 02:01:29 GMT
 */

export const PrimerFontStackSystem = "-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const PrimerFontStackSansSerif = "-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const PrimerFontStackMonospace = "ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace";
export const PrimerTextDisplayLineBoxHeight = "3.5rem";
export const PrimerTextDisplaySize = "2.5rem";
export const PrimerTextDisplayLineHeight = "calc(56/40)";
export const PrimerTextDisplayWeight = "500";
export const PrimerTextDisplayShorthand = {"fontWeight":"500","fontSize":"2.5rem","lineHeight":"calc(56/40)","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextTitleSizeLarge = "2rem";
export const PrimerTextTitleSizeMedium = "1.25rem";
export const PrimerTextTitleLineHeightLarge = "calc(48/32)";
export const PrimerTextTitleLineHeightMedium = "calc(32/20)";
export const PrimerTextTitleWeight = "600";
export const PrimerTextTitleShorthandLarge = {"fontWeight":"600","fontSize":"2rem","lineHeight":"calc(48/32)","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextTitleShorthandMedium = {"fontWeight":"600","fontSize":"1.25rem","lineHeight":"calc(32/20)","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextSubtitleSize = "1.25rem";
export const PrimerTextSubtitleLineHeight = "calc(32/20)";
export const PrimerTextSubtitleWeight = "400";
export const PrimerTextSubtitleShorthand = {"fontWeight":"400","fontSize":"1.25rem","lineHeight":"calc(32/20)","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextBodySizeLarge = "1rem";
export const PrimerTextBodySizeMedium = "0.875rem";
export const PrimerTextBodySizeSmall = "0.75rem";
export const PrimerTextBodyLineHeightLarge = "calc(24/16)";
export const PrimerTextBodyLineHeightMedium = "calc(20/14)";
export const PrimerTextBodyLineHeightSmall = "calc(20/12)";
export const PrimerTextBodyWeight = "400";
export const PrimerTextBodyShorthandLarge = {"fontWeight":"400","fontSize":"1rem","lineHeight":"calc(24/16)","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextBodyShorthandMedium = {"fontWeight":"400","fontSize":"0.875rem","lineHeight":"calc(20/14)","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextBodyShorthandSmall = {"fontWeight":"400","fontSize":"0.75rem","lineHeight":"calc(20/12)","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextCaptionSize = "0.75rem";
export const PrimerTextCaptionLineHeight = "calc(16/12)";
export const PrimerTextCaptionWeight = "400";
export const PrimerTextCaptionShorthand = {"fontWeight":"400","fontSize":"0.75rem","lineHeight":"calc(16/12)","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextCodeBlockSize = "0.8125rem";
export const PrimerTextCodeBlockLineHeight = "calc(20/13)";
export const PrimerTextCodeBlockWeight = "400";
export const PrimerTextCodeBlockShorthand = {"fontWeight":"400","fontSize":"0.8125rem","lineHeight":"calc(20/13)","fontFamily":"ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace"};
export const PrimerTextCodeInlineSize = "0.9285em";
export const PrimerTextCodeInlineWeight = "400";
export const PrimerTextCodeInlineShorthand = {"fontWeight":"400","fontSize":"0.9285em","fontFamily":"ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace"};
